### PR TITLE
initialRequest

### PR DESCRIPTION
**Pull request by noahSerafin**
- Component complete
- Snapshot testing done

### CODE REVIEW

File: src/components/HeaderImageVideoContainer/HeaderImageVideoContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+const HeaderImageVideoContainer = (props) => {
+  const { content, isVideo } = props;
+
+  if (!content) {
+    console.error("No props provided to header image/video container.");
+    return null;
+  }
+
+  return <div className="header-container">{isVideo ? <YoutubeEmbed embedId={content} /> : <img src={content} className="header-image" alt="Event image" />}</div>;
```
Comment (DanForder) [inline src/components/HeaderImageVideoContainer/HeaderImageVideoContainer.jsx:12]: classname should match name of component

File: src/components/HeaderImageVideoContainer/HeaderImageVideoContainer.test.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+test("it should render the component", () => {
+  const { container } = customRender(<HeaderImageVideoContainer content={placeHolderImage} isVideo={false} />);
+  expect(container).toMatchSnapshot();
+});
+
+test("it should not render anything without props", () => {
+  const { container } = customRender(<HeaderImageVideoContainer />);
+  expect(container).toMatchSnapshot();
+});
```
Comment (DanForder) [inline src/components/HeaderImageVideoContainer/HeaderImageVideoContainer.test.js:13]: add extra snapshot for isVideo true

**Review by DanForder — CHANGES_REQUESTED** [review]

File: src/components/HeaderImageVideoContainer/HeaderImageVideoContainer.test.js (hunk 2 above)
Comment (jasenscode) [reply src/components/HeaderImageVideoContainer/HeaderImageVideoContainer.test.js:18]: added

File: src/components/HeaderImageVideoContainer/HeaderImageVideoContainer.jsx (hunk 1 above)
Comment (jasenscode) [reply src/components/HeaderImageVideoContainer/HeaderImageVideoContainer.jsx]: amended

**Review by DanForder — APPROVED** [review]

**Review by DanForder — CHANGES_REQUESTED** [review]
fix merge conflicts

**Review by DanForder — APPROVED** [review]

**Pull request merged**